Infrastructure/arc_lambda/index.js

```diff
@@ -43,21 +45,82 @@ exports.handler = async(event, context, callback) => {
       output.on('end', function() {
         console.log('Data has been drained');
       });
-      
+      output.on('error', function (err) { 
+        console.log('[OUTPUT STREAM ERROR]', err); 
+      });
+
+      archive.on('error', function(err) {
+        console.log('[ARCHIVER ERROR]', err); 
```

Comment:
can multiple archive events be dispatched? what else can we do with the error other than logging to stderr? should we abort the response with a failure http status code?

Reply:
This is lambda function invoked by cron job. Http code would be ignored anyway :) 
I will add documentation that describes steps how we close intake (two cron jobs need to run - one marks applications as received, second generates the archive of attachments)